atd-etl/socrata_export/_compare_column_metadata.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+"""helper to compare socrata dataset metadata between staging and prod"""
+from utils.queries import QUERIES
+from utils.socrata import get_socrata_client
+
+datasets = [
+    {"name": "crashes", "staging": "3aut-fhzp", "prod": "y2wy-tgr5"},
+    {"name": "people", "staging": "v3x4-fjgm", "prod": "xecs-rpy9"},
+]
+
+
+def compare_col_arrays(list_a, list_b, a_list_name, b_list_name, primary_list="a"):
+    todos = []
+    for col in list_a:
+        key = col["fieldName"]
+        matching_col = None
+        try:
+            matching_col = next(
```

Comment:
this is a cool way to do this, lots of TILs in this python script for me :cowboy_hat_face: